projects/fw_102_homework/src/main.c

```diff
@@ -0,0 +1,39 @@
+#include <stdint.h>
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+typedef struct Counters {
+  uint8_t counter_a;
+  uint8_t counter_b;
+} Counters;
+
+void prv_timer_callback(SoftTimerId timer_id, void *context) {
```

Comment:
Since this is a private function, it should be marked `static`. That makes it private to a file, so other files can't see it. This is important in bigger projects because if you don't have `static`, then for example if some other file defined a `prv_timer_callback` function as well, then you might get linker errors.